### ISSUE TITLE
0 loss while training the CNN model

### ISSUE THREAD

**Issue by nithishk-03** [manthan89-py/Plant-Disease-Detection#8]
While training the CNN model in the batch gradient step this is  the result I'm getting
 Epoch : 1/5 Train_loss:0.000 Test_loss:0.000
I ran the code exactly, I didn't change anything.

**Comment by sravanijatoth02** [comment]
You have to change a code little bit  to get the correct output. 
step:
def batch_gd(model, criterion, train_loader, validation_loader, epochs)
 train_losses = np.zeros(epochs)
 validation_losses = np.zeros(epochs) 

Update this step ,then  the code runs perfectly  with 88% accuracy.

**Comment by manthan89-py** [comment]
@sravanijatoth02 Thanks for pointing it out. I have fixed this issue.
